werk24/models/property/refractive_index.py

```diff
@@ -42,29 +42,39 @@ class Config:
 
 
 class W24PropertyRefractiveVariationSchottGrade(W24PropertyRefractiveVariation):
+    """ Refractive Variation Schott Grade """
     variation_type: Literal["SCHOTT_GRADE"] = "SCHOTT_GRADE"
     blurb: str = Field(examples=("LN", "LH1", "LH2"))
     grade: str = Field(examples=["LN", "LH1", "LH2"])
 
 
 class W24PropertyRefractiveVariationIso12123(W24PropertyRefractiveVariation):
+    """ Refractive Variation ISO 12123 """
     variation_type: Literal["ISO_12123"] = "ISO_12123"
     blurb: str = Field(examples=("NV20", "NV10", "NV05"))
     grade: str = Field(examples=["NV20", "NV10", "NV05"])
 
 
+class W24PropertyRefractiveVariationFreeText(W24PropertyRefractiveVariation):
+    """ Refractive Variation Free Text """
+    free_text: str
```

Comment:
nee to have variation_type: Literal["FREETEXT"] = "FREETEXT"
Otherwise the deserializer doesnt know what to do

Reply:
updated